src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -170,5 +171,11 @@ public static bool IsMountEnabled(this IEnvironment environment)
             var mountEnabled = environment.GetEnvironmentVariable(MountEnabled);
             return !string.IsNullOrEmpty(mountEnabled) && string.Equals(mountEnabled, "1");
         }
+
+        private static bool IsValidZipSetting(string appSetting)
```

Comment:
Please add an EnvironmentExtensionsTests suite and add some [Theory] tests for this.

Reply:
@mathewc - done!